### DISCUSSION TITLE
Re-check repairs

### DISCUSSION THREAD

**Discussion by Protoncek** [frenck/spook#463] [Q&A] [answered]
Hi!
I just found and installed spook i immediately ran into one "case": spook reported invalid entity used in my automation - that part is correct, because i did some testing with tags (with esphome module) and i've had disabled automation with itag entity in it. Because i deleted that device from esphome entity was non-existing, which is true. So, i deleted automation, but repair warning keeps showing until i restart home assistant.  Is this expected behaviour? I restarted automations, but that didn't help...  i also tried to ignore and unignore, but also didn't help. 
Is there a way to trigger repairs check ?

**Comment by frenck** [comment, marked as answer]
> So, i deleted automation, but repair warning keeps showing until i restart home assistant. Is this expected behaviour

Yup.

**Reply by Protoncek** [reply]
OK, good to know. Thanks a lot! 
And thanks again for this addon!
